client/packages/invoices/src/Prescriptions/LineEditView/LineEditView.tsx

```diff
@@ -138,6 +138,7 @@ export const PrescriptionLineEditView = () => {
       );
     }
     setIsDirty(false);
+    setAllDraftLines({});
```

Comment:
Fix for #6061. Problem was the item ended up in the `allDraftLines` array twice -- once indexed by `"new"` and once indexed by its ID. Best to just start with a fresh object after a save (same as when you first load the page).